### PR TITLE
Updating External Import Paths

### PR DESCRIPTION
**Pull request by Dig-Doug**
Fixes issue #91.

### CODE REVIEW

**Comment by qiwzhang** [comment]
@Dig-Doug could you submit this change?   I need this fix.  Thanks.

**Comment by Dig-Doug** [comment]
I don't have write access so I cannot merge.

@pcj - Is there anything blocking this change from being merged?

**Comment by promiseofcake** [comment]
+1 :clap:

File: protobuf/internal/proto_compile.bzl [hunk 1]
```diff
@@ -389,6 +389,23 @@ def _get_external_root(ctx):
     return "."
 
 
+def _update_import_paths(ctx, builder):
```
Comment (guptasu) [inline protobuf/internal/proto_compile.bzl]: Shall we rename this to update_external_import_paths

File: protobuf/internal/proto_compile.bzl [hunk 2]
```diff
@@ -389,6 +389,23 @@ def _get_external_root(ctx):
...
+  """Updates import paths beginning with 'external' so that they point to external/."""
+  execdir = _get_external_root(ctx)
+  final_imports = []
+  for i in builder["imports"]:
```
Comment (guptasu) [inline protobuf/internal/proto_compile.bzl]: I->path

**Comment by pcj** [comment]
Applying change as-is, let's see if it needs further modification.  Sorry for awful delay on this @Dig-Doug!

**Comment by bocon13** [comment]
fixes #91

**Pull request merged**